test/HydraStaking/HydraStaking.test.ts

```diff
@@ -278,6 +278,36 @@ export function RunHydraStakingTests(): void {
           .to.emit(hydraStaking, "StakingRewardsClaimed")
           .withArgs(rewardingValidator.address, reward);
       });
+
+      // TODO: update _claimStakingRewards in VestedStaking so we can claim rewards from the normal position
```

Comment:
What about this TODO?

Reply:
Will be discussed and added to another ticket, right now it is in the review comments! Now the idea is not to change the logic.